src/uproot/behaviors/TBranch.py

```diff
@@ -501,196 +503,9 @@ def lazy(
     * :doc:`uproot.behaviors.TBranch.lazy` (this function): returns a lazily
       read array from ``TTrees``.
     """
-    awkward = uproot.extras.awkward()
-    files = uproot._util.regularize_files(files)
-    decompression_executor, interpretation_executor = _regularize_executors(
-        decompression_executor, interpretation_executor, None
+    raise NotImplementedError(
+        "uproot.lazy was deprecated in Uproot5 in favour of uproot.dask. If you still wish to use uproot.lazy please install Uproot4"
```

Comment:
```suggestion
        "uproot.lazy is removed in Uproot5 (only here for the deprecation message in the online docs)"
```

Reply:
I suppose this message assumes that the reader (the user) knows what version of Uproot they're using; that's not always the case.